### PR TITLE
Main screen updates

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -1408,21 +1401,22 @@
...
             "buy": "구입",
             "contacts": "연락처",
+            "convert": "Convert",
+            "deposit": "Deposit",
             "devers": "데버",
             "evers": "에버",
             "invite": "초대하기",
             "receive": "받기",
             "send": "보내기",
+            "showAssets": "Show assets",
```
Comment (vetkamak) [inline src/ko.json]: @kkkhesha could you translate it, please?

File: src/ru.json [hunk 2]
```diff
@@ -1406,21 +1399,22 @@
...
+            "buy": "Купить",
             "contacts": "Контакты",
+            "convert": "Convert",
+            "deposit": "Deposit",
+            "devers": "Дэверов",
+            "evers": "Эверов",
             "invite": "Пригласить",
             "receive": "Получить",
             "send": "Отправить",
+            "showAssets": "Show assets",
```
Comment (vetkamak) [inline src/ru.json]: @kkkhesha and english strings here as well

File: src/ko.json [hunk 3]
```diff
@@ -1408,21 +1401,22 @@
         },
         "main": {
             "activate": "활성화",
+            "add": "Add",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "add": "추가",
```

File: src/ko.json [hunk 4]
```diff
@@ -1408,21 +1401,22 @@
...
             "assets": "자산",
             "assetsBalance": "자산 잔액",
             "buy": "구입",
             "contacts": "연락처",
+            "convert": "Convert",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "convert": "전환",
```

File: src/ko.json [hunk 5]
```diff
@@ -1408,21 +1401,22 @@
...
+            "deposit": "Deposit",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "deposit": "예금",
```

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "showAssets": "주소 표시",
```

File: src/ru.json [hunk 6]
```diff
@@ -1406,21 +1399,22 @@
         },
         "main": {
             "activate": "Активировать",
+            "add": "Add",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
            "add": "Добавить",
```

File: src/ru.json [hunk 7]
```diff
@@ -1406,21 +1399,22 @@
...
             "assets": "Активы",
             "assetsBalance": "Баланс активов",
+            "buy": "Купить",
             "contacts": "Контакты",
+            "convert": "Convert",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
            "convert": "Конвертировать",
```

File: src/ru.json [hunk 8]
```diff
@@ -1406,21 +1399,22 @@
...
+            "deposit": "Deposit",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
            "deposit": "Внести",
```

File: src/ru.json (hunk 2 above)
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
            "showAssets": "Показать адрес",
```

**Review by sertony — APPROVED** [review]

**Review by lochmatij — APPROVED** [review]

**Pull request merged**